src/components/Clients/Clients.styles.ts

```diff
@@ -1,18 +1,27 @@
 import styled from 'styled-components'
 
-import Container from 'ui/Container/Container'
 import { breakpoints } from 'shared/breakpoints'
 import { colors } from 'styles/colors'
+import Section from 'ui/Section/Section'
 
-export const StyledClientsSection = styled.section`
+export const SectionClients = styled(Section)`
   box-shadow: 0 0 1.25rem 0 rgba(30, 26, 26, 0.5);
   background-color: ${colors.white.main};
-  width: 100%;
+  width: calc(100% + 3rem);
```

Comment:
add variable to explain better 3rem value

Reply:
Ready!